### PR TITLE
Replace Slack Inviter with a redirect app

### PR DESCRIPTION
**Pull request by feanil**
Delete the code of the existing app because we can no longer use this
mechanisim to invite people to our slack workspace. The api that this
app uses was never official and so it is no longer possible to make new
api tokens that have the correct permission to access this old
undocumented API.

We're gonna replace this app with a redirect to the slack invite link in
case the heroku app location is still hardcoded somewhere.

### CODE REVIEW

**Comment by feanil** [comment]
Current dependencies on/for this PR:
* master
  * **PR #4** <a href="https://app.graphite.dev/github/pr/openedx/openedx-slack-invite/4" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/openedx/openedx-slack-invite/4?utm_source=stack-comment).

File: app.js [hunk 1]
```diff
@@ -1,58 +1,12 @@
...
-        error: {}
-    });
-});
-
-
-module.exports = app;
+const express = require('express')
+const app = express()
+const port = process.env.PORT || 3000
+const redirect_url = process.env.REDIRECT_URL || "https://example.com"
```
Comment (Carlos-Muniz) [inline app.js]: Why are we using `"https://example.com"` as a default?

Comment (feanil) [reply app.js]: I could just hardcode the invite URL but I wanted to be able change it in the future so it's set via an environment variable in heroku.  I could see an argument of the app just not starting up if that setting is not set though.

Comment (Carlos-Muniz) [reply app.js]: I'm not sure whats the right thing to do here. But I think we would probably be more likely to hear from someone that they are getting an error than they are getting the wrong link.

**Review by Carlos-Muniz — APPROVED** [review]
Everything looks like it works as its supposed to. 

The app when run locally returns `Error: REDIRECT_URL environment variable is undefined.` As it's supposed to.

**Pull request merged**